clicmd

```diff
@@ -78,7 +78,7 @@ fi
 # by the name of the CLI command that called this function
 function exec_tool {
   local FNAME="${FUNCNAME[1]##cmd_}"
-  local PNAME="bmc ${FNAME//_/ }"
+  local PNAME="health ${FNAME//_/ }"
```

Comment:
This looks wrong, just like the original code.
I looks like the original code assumed that `exec_tool` was only called from `bmc.*` scripts, which is not true now.
It looks like `$(basename "${USER_CMD}")` must be used instead of `bmc` or `health`.